src/axom/quest/MarchingCubes.cpp

```diff
@@ -118,13 +126,16 @@ void MarchingCubes::populateContourMesh(
       auto* domainIdPtr =
         mesh.getFieldPtr<axom::IndexType>(domainIdField,
                                           axom::mint::CELL_CENTERED);
+
+      int userDomainId = single->getDomainId(dId);
+
       // TODO: Verify that UnstructuredMesh only supports host memory.
```

Comment:
Does this comment still need to be here?

Reply:
No.  Removing.